server/server.js

```diff
@@ -2,29 +2,78 @@ const express = require('express');
 const path = require('path');
 const bodyParser = require('body-parser');
 const cookieParser = require('cookie-parser');
-const quizerController = require('./controllers/quizController')
+
+const quizController = require('./controllers/quizController');
+const userController = require('./controllers/userController');
+const cookieController = require('./controllers/cookieController');
+const sessionController = require('./controllers/sessionController');
 
 const app = express();
+
+app.use(bodyParser.urlencoded({ extended: true }));
 app.use(express.json());
+app.use(cookieParser());
+
+app.use('/build', express.static(path.resolve(__dirname, '../build')));
+
+app.get('/', (req, res) => {
+  //get request to login/signup page
+  return res.status(200).sendFile(path.resolve(__dirname, '../index.html'));
+});
+
+app.post(
+  '/signup',
+  userController.createUser,
+  cookieController.setSSIDCookie,
+  sessionController.startSession,
+  (req, res) => {
+    // on failed signup, send boolean false
+    if (res.locals.alreadyExists)
+      return res
+        .status(200)
+        .json({ message: 'Username already taken!', loggedIn: false });
+    // on successful signup, send boolean true
+    res.status(200).json({ message: 'New user added!', loggedIn: true });
```

Comment:
I was told in my APC to always start this with `return` as best practice.

Reply:
Agreed, updated.

Reply:
Looks good.